### ISSUE TITLE
#10 More details about MSVD on zero-example video retrieval

### ISSUE THREAD

**Issue by sharontaozi** [danieljf24/dual_encoding#11]
In the paper, "So we assess the models previously trained on MSR-VTT using the MSVD test set" refers to training with the entire data set of MSRVTT, and testing the model with the test set of MSVD（670 vidoes）? Or the test model is to use the entire MSVD dataset（1970 vidoes） as a test set？

**Comment by danieljf24** [comment]
Sorry for the confusion. Indeed, we directly used the Dual encoding model in Table 1 without any finetune, so the model was trained in the training set (6,513 video clips) of MSRVTT. Besides, we evaluated the model on the test set (670 video clips) of MSVD.

**Comment by sharontaozi** [comment]
Thanks!After I read your other paper 《Word2VisualVec: Image and Video to Sentence Matching by Visual Feature Prediction》，I have some questions, and I wrote on https://github.com/danieljf24/w2vv/issues/11
I really hope you can tell me more details.Thank you!

**Comment by xixiareone** [comment]
Would you like to evaluate all sentences in the MSVD dataset, especially in the test phase, or just randomly select 5 sentences from the MSVD?

thank you very much!

**Comment by xixiareone** [comment]
I would like to ask you a question: in the MSVD data set, especially in the test phase, do you evaluate all sentences, or just randomly select 5 sentences from the MSVD for evaluation?

**Comment by danieljf24** [comment]
On MSVD, we tested on the test set of 670 video clips, and using all the corresponding sentences.
It is worth noting that in order to evaluate the generalization ability of our proposed model, the results on MVSD were obtained using the model trained in the training set (6,513 video clips) of MSRVTT.

**Comment by danieljf24** [comment]
In the previous version of our w2vv [paper](https://arxiv.org/pdf/1604.06838.pdf), in Table 5, we used all the corresponding sentences instead of randomly sampled 5 sentences for each test video ( Results using data partition from Xu et al. [40]). For the results using data partition from Otani et al. [24], we used  5 sentences for each test video provided by Otani et al, while all the training sentences.  1,200, 100 and 670 video clips for training, validation, and test.

**Comment by danieljf24** [comment]
The MSVD data with extracted video features have been updated. https://github.com/danieljf24/dual_encoding#dual-encoding-on-msvd

**Comment by xixiareone** [comment]
Thank you very much！！！





---Original---

**Comment by xixiareone** [comment]
I  am  sorry to bother you, can you share the entails of code in MSVD？

**Comment by sharontaozi** [comment]
Thank you!!

**Comment by xixiareone** [comment]
> 谢谢！！

Can we communicate with each other? Because I'm also from China.

**Comment by sharontaozi** [comment]
@xixiareone @danieljf24 can we add QQ or wechat? (ID:499400402)

**Comment by xixiareone** [comment]
yeap~I will add you soon




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"sharontaozi"<notifications@github.com&gt;;
发送时间:&nbsp;2020年5月30日(星期六) 中午11:29
收件人:&nbsp;"danieljf24/dual_encoding"<dual_encoding@noreply.github.com&gt;;
抄送:&nbsp;"Gin"<xixiareone@foxmail.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [danieljf24/dual_encoding] #10 More details about MSVD on zero-example video retrieval (#11)





 
@xixiareone @danieljf24 can we add QQ or wechat? (ID:499400402)

**Comment by danieljf24** [comment]
> I am sorry to bother you, can you share the entails of code in MSVD？

The corresponding code has also been released.
